applications/IgaApplication/custom_elements/shell_5p_element.cpp

```diff
@@ -52,28 +48,10 @@ namespace Kratos
             reference_Curvature[point_number] = kinematic_variables.curvature;
             reference_TransShear[point_number] = kinematic_variables.transShear;
         }
-        InitializeMaterial();
-        KRATOS_CATCH("")
-    }
-
-    void Shell5pElement::InitializeMaterial()
-    {
-        KRATOS_TRY
-            const GeometryType& r_geometry = GetGeometry();
-        const Properties& r_properties = GetProperties();
-
-        const SizeType r_number_of_integration_points = r_geometry.IntegrationPointsNumber();
-
-        //Constitutive Law initialisation
-        mConstitutiveLawVector.resize(r_number_of_integration_points);
-
-        std::fill(mConstitutiveLawVector.begin(), mConstitutiveLawVector.end(), GetProperties()[CONSTITUTIVE_LAW]->Clone());
-
-        for (IndexType point_number = 0; point_number < mConstitutiveLawVector.size(); ++point_number)
-            mConstitutiveLawVector[point_number]->InitializeMaterial(r_properties, r_geometry, row(m_N, point_number));
 
```

Comment:
Ah ok i didn't know if an issue occurs if this is not initialized. But I'm happy with every removal of useless lines of code!

Reply:
well, basically as this CL is not considering a constitutive law I thought it would rather confuse people. Additionally, it releases quite some memory.